coderd/templates.go

```diff
@@ -351,6 +352,15 @@ func (api *API) postTemplateByOrganization(rw http.ResponseWriter, r *http.Reque
 		}
 	}
 
+	if createTemplate.CORSBehavior != nil && *createTemplate.CORSBehavior != "" {
+		val := createTemplate.CORSBehavior
+		if err := val.Validate(); err != nil {
+			validErrs = append(validErrs, codersdk.ValidationError{Field: "cors_behavior", Detail: err.Error()})
+		} else {
+			corsBehavior = database.CorsBehavior(*val)
+		}
```

Comment:
Instead of making a `Validate` function, we can use the database `Valid` function. It's less to keep in sync between the database and the sdk
```suggestion
		val := database.CorsBehavior(*createTemplate.CORSBehavior)
		if !val.Valid() {
			validErrs = append(validErrs, codersdk.ValidationError{Field: "cors_behavior",
				Detail: fmt.Sprintf("Invalid CORS behavior %q. Must be one of [%s]", *createTemplate.CORSBehavior, strings.Join(slice.ToStrings(database.AllCorsBehaviorValues()), ", ")),
			})
		} else {
			corsBehavior = val
		}
```